### PR TITLE
Camouflage marines

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Can you spot the 8 marines in this picture? Research states that 80% of SOM terrorists, and 65% of xenomorphs cannot. Are YOU a terrorist or alien?
image

Makes xenonaut armour spawn with it's coloured variants based on the ground map, like jaeger does, to grossly buff marines by rendering them invisible to the naked eye through the miracle of camouflage. You can still change the colour manually of course.

Additionally added desert camo as a default for Chigusa.

Also made xenonauten hats not a child of jaeger helmets as they're not greyscale/hyperscale so never should have been, and they were inheriting a bunch of random shit they didn't need to.

## Why It's Good For The Game
Marines in need of a huge buff via advanced camouflage

## Changelog

:cl:
add: Xenonauten armour spawns with map appropriate colours where applicable
/:cl:

### CODE REVIEW

**Comment by Naaanii** [comment]
GUERILLA WARFARE! LET'S GOOOO!!!

**Comment by Losenis** [comment]
> Black drip nerf

Disgusting

**Comment by SilencedMP5A5** [comment]
this image is gaslighting me

_where are the ungas oh god oh fuck_

**Comment by MereWalker** [comment]
I genuinely can't see it. Please where is my giant red circle.

**Comment by Strayed-Raven** [comment]
black drip nerf.

**Comment by UntoldTactics** [comment]
I mean okay but I hope that image is a joke. Xenos have literal thermal vision after all, and idk how your making marines register at -5 degrees celcius or whatever

**Comment by MereWalker** [comment]
Please! I need the red circle! I've been looking for DAYS now. At least tell me they are real!

**Review by Kuro020 — APPROVED** [review]

File: code/datums/loadout/old_typepaths.dm [hunk 1]
```diff
@@ -5,6 +5,6 @@
 
 
 /obj/item/clothing/suit/modular/pas11x
-/obj/item/clothing/head/modular/marine/m10x/tech
-/obj/item/clothing/head/modular/marine/m10x/corpsman
-/obj/item/clothing/head/modular/marine/m10x/standard
+/obj/item/clothing/head/modular/m10x/tech
+/obj/item/clothing/head/modular/m10x/corpsman
+/obj/item/clothing/head/modular/m10x/standard
```
Comment (TiviPlus) [inline code/datums/loadout/old_typepaths.dm:10]: read the comment in this file pls

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/datums/loadout/old_typepaths.dm (hunk 1 above)
Comment (Lumipharon) [reply code/datums/loadout/old_typepaths.dm:10]: Reading is for nerds.

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Wont this break loadouts meaning you want to convert old ones or did you discuss that w/ brave

**Comment by Lumipharon** [comment]
> Wont this break loadouts meaning you want to convert old ones or did you discuss that w/ brave

I talked with brave. Considering we're breaking Jaeger fundamentally so those loadouts will have to be remade, I thought it better to just have this at the same time and same situation

**Pull request merged**